notebooks/benchmark/Default Mask Benchmark.ipynb

```diff
@@ -0,0 +1,283 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "compare_plot(benchmarks)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3.7.9 64-bit",
```

Comment:
we should make sure metadata is consistent across all shap notebooks (to keep things clean)

Reply:
Will change the kernel to be Python 3 (same as all the rest of the notebooks within shap)